src/java.base/share/classes/java/lang/module/ModuleDescriptor.java

```diff
@@ -1516,7 +1516,7 @@ public Optional<String> mainClass() {
     }
 
     /**
-     * Returns the set of packages in the module.
+     * Returns the set of all packages in the module.
```

Comment:
We changed this from "Returns the names of all packages in this module" to "Returns the set of packages in the module" as part of JDK-8173393 in JDK 9 [1]. This was a large update and I can't recall if there was a reason to drop "all" at the time. Looking at the API docs now, I think it's useful to say "all packages".  It can use `{@return ..}` as Chen suggests. I think it might be clearer if we just dropped the second paragraph, it's not needed. (I think this second paragraph may date when when there was a method to get the concealed/non-exported packages).

[1] https://hg.openjdk.org/jdk9/jdk9/jdk/rev/1c8f054f6278

Reply:
Thank you Alan for finding this history. I have now updated the PR to remove the second paragraph and also follow Chen's and your suggestion about the `{@return ...}` usage.